webapp/src/app/pacman-features/secondary-components/policy-violation-desc/policy-violation-desc.component.html

```diff
@@ -21,7 +21,13 @@
             {{violationData.policyViolated}}
         </div>
     </div>
-    <div class="policy-violation-label flex flex-col" *ngIf="violationData.policyDescription">
+    <div class="policy-violation-label flex flex-col" *ngIf="Params.issueId">
+        <div class="policy-violation-label-header capitalize">
+            violation ID</div>
+        <div class="policy-violation-label-content" [innerHTML]="Params.issueId" title="{{Params.issueId}}">
```

Comment:
if `Params.issueId` doesn't contain any [entities](https://developer.mozilla.org/en-US/docs/Glossary/Entity), than you shouldn't use [innerHTML] property binding, use instead one-way binding, eg. `<div class="policy-violation-label-content" ...>{{ params.issueId }}</div>`